com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Artistic/Normal/NormalFromTextureNode.cs

```diff
@@ -13,7 +13,7 @@ internal class NormalFromTextureNode : IStandardNode
             Name,
             functions: new FunctionDescriptor[] {
                 new(
-                    "NormalFromTexture3Samples",
+                    "3Samples",
```

Comment:
Leading with a number on a function name could be problematic.

Reply:
Fixed